### PR TITLE
Split version increments into related and unrelated to the API

### PR DESCRIPTION
**Pull request by N-Nagorny**

### CODE REVIEW

File: docs/Interoperability.md [hunk 1]
```diff
@@ -55,6 +55,6 @@ The UUIDs used to advertise Senders and Receivers in the Flow Compatibility Mana
 
 ## Version Increments
 
-In order to prevent unnecessary polling of the Flow Compatibility Management API, changes to Inputs and Outputs are signalled via the IS-04 versioning mechanism. When the Input/Output parameters of a Sender/Receiver are modified, when set of Inputs/Outputs associated with the Sender/Receiver is changed, or when Status of Sender/Receiver is updated, then the `version` attribute of the relevant IS-04 Sender/Receiver MUST be incremented.
+In order to prevent unnecessary polling of the Flow Compatibility Management API, changes to it are signalled via the IS-04 versioning mechanism. E.g. when the Input/Output parameters of a Sender/Receiver are modified, when set of Inputs/Outputs associated with the Sender/Receiver is changed, or when Status of Sender/Receiver is updated, then the `version` attribute of the relevant IS-04 Sender/Receiver MUST be incremented.
```
Comment (alabou) [inline docs/Interoperability.md]: I would not  use "changes to it" ... but simply "changes"

I would add "when state of Inputs/Outputs associated with the Sender/Receiver changes"

I would rephrase "In order to prevent unnecessary polling of the Flow Compatibility Management API, changes are signaled via the IS-04 versioning mechanism of the associated Sender and Receiver objects"

File: docs/Interoperability.md [hunk 2]
```diff
@@ -55,6 +55,6 @@ The UUIDs used to advertise Senders and Receivers in the Flow Compatibility Mana
...
 
-If `PUT /constraints/active`, `DELETE /constraints/active` or `PUT /edid/base` operation applied to a Sender causes a change of Inputs, Source or Flow, their versions MUST be incremented as well. If such a change causes changes to the Connection API `/transportfile` resource, version of the relevant IS-04 Sender MUST be incremented.
+If any resource such as Source, Flow, Sender etc. changes over its lifetime, then its `version` attribute MUST be incremented as well.
```
Comment (alabou) [inline docs/Interoperability.md]: I would rephrase "If the state of any resource such as Source, Flow, Sender, Receiver, etc. changes over its lifetime, then its `version` attribute MUST be incremented as well."

File: docs/Interoperability.md (hunk 1 above)
Comment (N-Nagorny) [reply docs/Interoperability.md]: Thanks, it sounds better.

```suggestion
In order to prevent unnecessary polling of the Flow Compatibility Management API, changes are signaled via the IS-04 versioning mechanism of the associated Sender and Receiver resources. E.g. when the Input/Output parameters of a Sender/Receiver are modified, when set of Inputs/Outputs associated with the Sender/Receiver is changed, or when Status of Sender/Receiver is updated, then the `version` attribute of the relevant IS-04 Sender/Receiver MUST be incremented.
```

File: docs/Interoperability.md (hunk 2 above)
Comment (N-Nagorny) [reply docs/Interoperability.md]: It seems more confusing to me because we use term "state" for Sender/Receiver. What about "properties"?

```suggestion
If properties of any resource such as Source, Flow, Sender, Receiver, etc. change over its lifetime, then its `version` attribute MUST be incremented as well.
```

File: docs/Interoperability.md (hunk 1 above)
Comment (alabou) [reply docs/Interoperability.md]: Maybe "properties" instead of "parameters" ... ?

File: docs/Interoperability.md (hunk 2 above)
Comment (alabou) [reply docs/Interoperability.md]: ok

File: docs/Interoperability.md (hunk 1 above)
Comment (N-Nagorny) [reply docs/Interoperability.md]:
```suggestion
In order to prevent unnecessary polling of the Flow Compatibility Management API, changes are signaled via the IS-04 versioning mechanism of the associated Sender and Receiver resources. E.g. when properties of any Input/Output associated with the Sender/Receiver are modified, when set of Inputs/Outputs associated with the Sender/Receiver is changed, or when Status of Sender/Receiver is updated, then the `version` attribute of the relevant IS-04 Sender/Receiver MUST be incremented.
```
